### ISSUE TITLE
Plan for migrating to Lumos ? 

### ISSUE THREAD

**Issue by pbrinette** [LLNL/slapi#7]
Hello Team,

This is rather a discussion than an issue.

First thanks for your tool. We are using it (too) oftenly on our 2 Tfinity to gather the logs when we have trouble. 

We will upgrade the Tfinity to LumOS some and get rid of Bluscale and Windows CE.  

Do you plan to make a port of your tool for Lumos ?

Thanks.

**Comment by hawartens** [comment]
Hi. Yes. We have plans to update slapi to work with LumOS however there will be no compatibility expectations between the BlueScale version and the newer LumOS version. The new tool may use the same verb for something and the output will not be the same as the old tool. We don’t currently have a timeline for when we will have a version we can post here, but we will are definitely going to.

**Comment by pbrinette** [comment]
Hi Herb,
I currently have a remote access to a lumos library in spectra labs.
It seems that everything on the library is exposed on the json API. I can get the inventory with a simple curl.
Maybe the port would be easier.
Thanks. 
Pierre

**Comment by hawartens** [comment]
Yes. We also had access to that library as well. I have an initial version of slapi talking to LumOS with a couple commands pulling some JSON data from the library. We also have a local test library at LLNL running LumOS that we will be developing against. I just want make sure to get all the structures generated and defined properly based on the defined JSON API (and in case the api changes which we have already seen once since LumOS is still in beta). That way hopefully there will be minimal changes needed for slapi if/when that does happen.

**Comment by jeffderbyshire** [comment]
Hello from FNAL, any updates on slapi working with LumOS?  Does LumOS have documentation on its REST API ?

**Comment by hawartens** [comment]
> Hello from FNAL, any updates on slapi working with LumOS?  Does LumOS have documentation on its REST API ?

We have a working version of SLAPI with LumOS. We are planning to open source our version and share it with anyone who wants it. We just need to go through the process to make it available for all.

**Comment by hawartens** [comment]
We have released our initial version of slapi for LumOS. Hopefully it works well for you. Sorry for any rough edges. We are happy continue to improve it over time.
